app/Controllers/feedController.php

```diff
@@ -571,6 +572,15 @@ public function actualizeAction() {
 			Minz_Request::setGoodNotification(_t('feedback.sub.feed.actualizeds'));
 			// No layout in ajax request.
 			$this->view->_layout(false);
+		} if ($noredirect) {
```

Comment:
It seems that it should be an `elseif` instead of an `if`. Using an `if` here change the flow and the following `else` is not linked anymore to the original `if`.
If the `if` is intended then the layout of the code is not properly done. There must be a blank line between statements.